src/main/java/org/prebid/server/proto/openrtb/ext/request/ExtRequestTargeting.java

```diff
@@ -16,6 +17,11 @@
      */
     JsonNode pricegranularity;
 
+    /**
+     * Defines the contract for bidrequest.ext.prebid.targeting.currency
+     */
+    Currency currency;
```

Comment:
I don't think this is right. `currency` object in request extension has different format than currency conversion rates published on the prebid.org site. I would suggest creating `ExtCurrency` class for use in request extension.